manifests/vhost.pp

```diff
@@ -397,6 +405,21 @@
     }
   }
 
+  if $modsec_audit_log_file {
```

Comment:
My intention was that you only need to set one attribute to enable logging.

I've changed the code a bit to improve this.